code/game/jobs/access.dm

```diff
@@ -388,7 +398,7 @@
 	return all_jobs
 
 /proc/get_all_centcom_jobs()
-	return list("VIP Guest","Custodian","Thunderdome Overseer","Emergency Response Team Member","Emergency Response Team Leader","Intel Officer","Medical Officer","Deathsquad Commando","Research Officer","Special Operations Officer","Nanotrasen Navy Representative","Nanotrasen Navy Officer","Nanotrasen Navy Captain","Supreme Commander")
+	return list("VIP Guest","Custodian","Thunderdome Overseer","Intel Officer","Medical Officer","Deathsquad Commando","Research Officer","Special Operations Officer","Nanotrasen Navy Representative","Nanotrasen Navy Officer","Nanotrasen Navy Captain","Supreme Commander")
 
```

Comment:
Editing this will leave ERT memebers out some checks elsewhere. See `code\datums\cache\crew.dm:28` for example, medical sensor list will no longer bold ERT.

Reply:
manually bolded now in the same way the magi and ntr are. they can't keep the generic centcom job since that overwrites the HUD icon to the generic CC one.

other edge cases should be fixed